assets/src/scripts/web-services/nldi-data.js

```diff
@@ -29,21 +29,26 @@ const fetchNldiData = function(nldiQuery, siteno) {
 
 // nldi webservice calls
 export const fetchNldiUpstreamSites = function(siteno) {
-    const upstreamSitesQuery = nldiUrl + '/'+ featureSource + '/' + usgs + siteno + '/navigate/' + upstreamNavigation + '/' + dataSource + distanceParam;
+    const upstreamSitesQuery = nldiUrl + '/' + featureSource + '/' + usgs + siteno + '/navigate/' + upstreamNavigation + '/' + dataSource + distanceParam;
```

Comment:
It would be lovely to change these to using template literals as long as you are in here. See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals